modules/virt-access-configuration-considerations.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-ssh.adoc
+
+:_content-type: REFERENCE
+[id="virt-access-configuration-considerations_{context}"]
+= Access configuration considerations
+
+Each method for configuring access to a virtual machine (VM) has advantages and limitations, depending on the traffic load and client requirements.
+
+Virtctl is recommended for troubleshooting VMs because it is simple to configure.
+
+Services provide excellent performance and are recommended for applications that are accessed from outside the cluster.
+
+If the internal cluster network cannot handle the traffic load, you can configure a secondary network.
+
+SSH key pair and virtctl::
+* Simple to configure.
+* Not recommended for high-traffic applications.
+* The API server must be able to handle the traffic load.
+* The clients must be able to access the API server.
+* The clients must have access credentials for the cluster.
+
+Cluster IP service::
+* The internal cluster network must be able to handle the traffic load.
+* The clients must be able to access an internal cluster IP address.
+
+Node port service::
+* The internal cluster network must be able to handle the traffic load.
+* The clients must be able to access at least one node.
+
+Load balancer service::
+* Each node must be able to handle the traffic load of one or more load balancer services.
+* MetalLB in layer 2 mode has the following limitations:
+** All traffic for a service IP address is routed through a single node, which might cause a bottleneck.
+** Failover might be very slow.
+* The clients must be able to access the node by using this network interface.
+
+Secondary network::
+* Excellent performance because traffic does not go through the internal cluster network.
+* VMs not exposed outside the cluster.
```

Comment:
This depends on the network configuration. It can be isolated within the cluster, or it can be routable from the outside.  We could say that this allows for flexible configuration of the network topology